pkg/alertmanager/amcfg_test.go

```diff
@@ -2885,7 +2885,66 @@ func TestTimeInterval(t *testing.T) {
 		})
 	}
 }
+func TestSanitizePushoverReceiverConfig(t *testing.T) {
+	logger := log.NewNopLogger()
+
+	for _, tc := range []struct {
+		name           string
+		againstVersion semver.Version
+		in             *alertmanagerConfig
+		expect         alertmanagerConfig
+		expectErr      bool
+	}{
```

Comment:
I noticed that there is only one test case, do you have any motivation to create a table test even if you're looping over a single test?

Reply:
Done.  add unittest case